griptape/drivers/prompt/amazon_bedrock_prompt_driver.py

```diff
@@ -29,16 +30,40 @@ def try_run(self, prompt_stack: PromptStack) -> TextArtifact:
         if isinstance(model_input, dict):
             payload.update(model_input)
 
-        response = self.bedrock_client.invoke_model(
-            modelId=self.model,
-            contentType='application/json',
-            accept='application/json',
-            body=json.dumps(payload),
-        )
+        if self.stream:
+            response = self.bedrock_client.invoke_model_with_response_stream(
```

Comment:
Make sure that this design also works for Cohere models. To enable streaming with Cohere command text model via Bedrock, one need to also pass a `"stream": true` parameter in the body as explained in https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html#model-parameters-cohere

Reply:
We actually didn't have access to Cohere until a few days ago so we're missing a Prompt Model Driver for it. I created a separate [issue](https://github.com/griptape-ai/griptape/issues/312) for it. Depending on timing I may implement streaming on that PR.